server/src/controllers/onNewTask.ts

```diff
@@ -11,7 +11,7 @@ const onNewTask = (io: socketio.Server, socket: socketio.Socket) => ({ roomId, t
 
   if (typeof room === 'string') return console.error(room);
 
-  room.task = task;
+  room.task.task = task;
```

Comment:
Tu tego tak nie mozna zostawiac, w ten sposob zmieniasz tekst taska a co z timestampem? Jezeli pojawia sie nowy task to timer powinien sie odswiezyc